src/Mastodon.elm

```diff
@@ -2,6 +2,7 @@ module Mastodon
     exposing
         ( AccessTokenResult
         , Account
+        , AccountId
```

Comment:
Hmm, I don't see the real benefit of exposing type aliases like this. In my views they're useful internally to this module, but that may just be me.